src/main/resources/templates/db_scripts/schema.json

```diff
@@ -398,14 +398,14 @@
         {
           "fieldName": "payment",
           "multiFieldNames": "amount, fromFundId, sourceInvoiceId, sourceInvoiceLineId, transactionType, expenseClassId",
-          "tOps": "DELETE",
-          "whereClause": "WHERE jsonb->>'transactionType' = 'Payment' OR jsonb->>'transactionType' = 'Pending payment'"
+          "tOps": "ADD",
+          "whereClause": "WHERE (jsonb->>'transactionType' = 'Payment' OR jsonb->>'transactionType' = 'Pending payment') AND (jsonb ->> 'sourceInvoiceLineId') IS NOT NULL"
```

Comment:
thinking that better to create new index for whereClause for field sourceInvoiceLineId.
We can use this example as we have for trasactionType:
<img width="743" height="696" alt="image" src="https://github.com/user-attachments/assets/0ee05624-0730-419c-adb8-62e64aed3853" />